pkg/webhook/tls/tls.go

```diff
@@ -0,0 +1,95 @@
+package tls
```

Comment:
Is this copied from somewhere or did you make it?

If just copied, please reference the source. Otherwise, please provide a good docs to this module. It may be reused and also, TLS is confusing.

Reply:
Is new stuff I did, will add some docs, we may want to also put it ouside of knmstate in case is resued by kubemacpool.